packages/library-sync/src/commonMain/kotlin/io/realm/kotlin/mongodb/auth/ApiKey.kt

```diff
@@ -0,0 +1,5 @@
+package io.realm.kotlin.mongodb.auth
+
+import io.realm.kotlin.types.ObjectId
+
+public data class ApiKey public constructor(public val id: ObjectId, public val value: String?, public val name: String, public val enabled: Boolean)
```

Comment:
Missing documentation of class and each parameter (since they are also class properties)

Reply:
The constructor parameters could be formatted in multiple lines to faciltate reading.